code/design_pattern/src/adapter/Adaptor.java

```diff
@@ -0,0 +1,22 @@
+package Adapter;
```

Comment:
Since this design pattern requires so many files, I would extract them into a directory (preferably called `java`)